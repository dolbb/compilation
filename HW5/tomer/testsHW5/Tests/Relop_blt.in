void main() {
	bool c1 = 3<1;
	if (c1) { 
		print("1\n");
	}
	else 
		print("else 1\n");
	//----------------
	bool d1 = 1<1;
	if (d1) { 
		print("2\n");
	}
	else 
		print("else 2\n");
	//----------------
	bool d12 = 1<3;
	if (d12) { 
		print("3\n");
	}
	else 
		print("else 3\n");
	//----------------
	//-----------------------------------
	bool e13;
	e13 = 3<1;
	if (e13) { 
		print("4\n");
	}
	else 
		print("else 4\n");
	//----------------
	bool f1;
	f1 = 1<1;
	if (f1) { 
		print("5\n");
	}
	else 
		print("else 5\n");
	//----------------
	bool f15;
	f15 = 1<3;
	if (f15) { 
		print("6\n");
	}
	else 
		print("else 6\n");
}