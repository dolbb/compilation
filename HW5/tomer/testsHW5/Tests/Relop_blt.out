Loaded: ./exceptions.s
else 1
else 2
3
else 4
else 5
6
